schema.sql

```diff
@@ -4,4 +4,7 @@
 
 -- DROP DATABASE IF EXISTS vet_clinic;
 
-CREATE TABLE animals(id INT PRIMARY NOT NULL, name VARCHAR(255), date_of_birth DATE, escape_attempts INT, neutered BOOLEAN, weight_kg DECIMAL(4,0));
\ No newline at end of file
+CREATE TABLE animals(id INT PRIMARY NOT NULL, name VARCHAR(255), date_of_birth DATE, escape_attempts INT, neutered BOOLEAN, weight_kg DECIMAL(4,0));
+CREATE TABLE owners (id serial PRIMARY KEY, full_name varchar(255), age integer);
+CREATE TABLE species (id serial PRIMARY KEY, name varchar(255));
+ALTER TABLE animals ADD COLUMN species_id integer, ADD COLUMN owner_id integer, ADD PRIMARY KEY (id), ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species (id), ADD CONSTRAINT fk_owner FOREIGN KEY (owner_id) REFERENCES owners (id);
```

Comment:
- [ ] Great work adding the columns `species_id` and `owner_id` to the `animals` table :clap: However it also requested in the project requirement to remove the `species` column from the `animals` table. Kindly add the code for removing the `species` column to the `schema.sql` file.
![Screenshot from 2023-09-22 08-18-58](https://github.com/boniface193/postgres/assets/91142432/fe5a8136-2bc5-4e7d-9913-67f558ed7c3b)

[OPTIOINAL]
- [ ] The column `id` it is already in the `animals` table when you created it in the previous activity, so no need to add it again